backend/question-service/lambda/app/scraper/.env.example

```diff
@@ -0,0 +1,3 @@
+MONGO_CONNECTION_STRING=mongodb://localhost:27017
```

Comment:
i think for this you should put the db service name
i.e. `mongodb://question-service-db:27017`

Reply:
Did not change:
- The scraper has to be run separately from the docker container. So it can't connect to the container with 'question-service-db'
- I could add it to docker-compose to allow it to connect to the network, but we don't want the scraper to run in local development unless opted into by the user
- Thus, they have to connect to the DB with `localhost:27017`